### PR TITLE
Objects Automated Test - LPS-141606 | master

### PR DESCRIPTION
**Pull request by andreefarias**
https://issues.liferay.com/browse/LPS-141606

### CODE REVIEW

**Comment by andreefarias** [comment]
ci:test:sf

File: modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase [hunk 1]
```diff
@@ -3145,12 +3145,79 @@ definition {
...
+		CreateObject.selectPanelCategoryKey(panelCategoryKey = "Control Panel > Object");
+
+		CreateObject.saveObject();
+
+		ObjectAdmin.openObjectAdmin();
 
-		// TODO LPS-141606 CanUpdateObjectWithUpdatePermission pending implementation
+		ObjectPortlet.viewEntry(entry = "company");
 
+		ObjectAdmin.viewCustomObject(name = "Object Edit");
```
Comment (evanilson-santana) [inline modules/apps/object/object-test/src/testFunctional/tests/CreateObject.testcase:3220]: Hi @andreefarias 

You can validate that the Title Field and Panel Category Key are updated too?

**Review by evanilson-santana — CHANGES_REQUESTED** [review]
Hi @andreefarias 

Please, check my comments and make the necessary changes.

Thanks!!!

**Review by evanilson-santana — APPROVED** [review]

**Comment by evanilson-santana** [comment]
ci:forward

**Comment by rodrigocunhaa** [comment]
ci:forward

**Comment by rodrigocunhaa** [comment]
Resent here: https://github.com/rodrigocunhaa/liferay-portal/pull/1452

**Pull request closed without merging**